kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/source/state/S3SourceBuilder.scala

```diff
@@ -42,7 +42,7 @@ object S3SourceState {
       connectorTaskId  <- IO.fromEither(ConnectorTaskId.fromProps(props))
       config           <- IO.fromEither(S3SourceConfig.fromProps(props))
       s3Client         <- IO.fromEither(AwsS3ClientCreator.make(config.s3Config))
-      storageInterface <- IO.delay(new AwsS3StorageInterface(connectorTaskId, s3Client))
+      storageInterface <- IO.delay(new AwsS3StorageInterface(connectorTaskId, s3Client, true))
```

Comment:
For the source, this is always true?

Reply:
The Index Manager invokes the `deleteFiles` method and is only used in the sink functionality; therefore, the `batchDelete` parameter is redundant for the source pathway. I have set it to the default `true`, but it isn't needed for the source pathway. I could put the default value in the `AwsS3StorageInterface` constructor to `true` and not set the value here.